### ISSUE TITLE
Useragent detected as Robot is actually not a Robot

### ISSUE THREAD

**Issue by evelzi** [nielsbasjes/yauaa#848]
**The exact useragent**

1. useragent: 'Mozilla/5.0 (Linux; Android 10; ANA-LX4) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/86.0.4240.110 Mobile Safari/537.36/quickapp))'
2. useragent: 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/100.0.4896.88 Safari/537.36/AbXJL0Fq5KNyln588U7'
3. useragent: 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/103.0.0.0 Safari/537.36/EAWinDTe5'
4. useragent: 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/107.0.0.0 Safari/537.36 Edg/107.0.1418.56/UHYlQ2yl-59'
5. useragent: 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/108.0.0.0 Safari/537.36/8vGeL2cj-43'

These are a few examples, but we have lots of similar cases. All of them end with *'/some-string'*  after the version.

**Incorrect fields**
DeviceClass: 'Robot'

**Expected behavior**
These cases are **not Robot**. We can assure this because they are users **who really buy on our platform**.

**Additional information**

- In all these cases, if we remove the last part ('/some-string'), they are not recognized as Robot anymore. 

- This behavior started from the v7.0.4.

**Comment by nielsbasjes** [comment]
Thanks for the report.
I was in doubt about these what they really are. Given these trailing parts I assumed non human. You have good indications they are human. Thanks, I didn't have that before.

Do you have any idea what causes these trailing parts? Perhaps some kind of anti fingerprinting plug-in?

**Comment by evelzi** [comment]
Nop, we don't have extra info. We searched for that on internet but couldn't find anything about those "extra part" at the end of the useragent.

Do you think it possible a change to support this?

**Comment by nielsbasjes** [comment]
I'm going to investigate this further.
I think marking it as anonymized yet still detect all the values is the best route for these.

**Comment by cengler** [comment]
We are looking for an extension or something that adds this suffix but we haven't found it yet

On the other hand, on this page that we use: 
https://developers.whatismybrowser.com/useragents/parse/#parse-useragent
Does some kind of cut in the userAgent when we analyze these cases

Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/107.0.0.0 Safari/537.36/r2vvqBpk-59
analyzes it as
Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/107.0.0.0 Safari/537.36
We can't find why they do it. And if they do it for some specific reason

If we can find something else, we'll let you know.

**Comment by nielsbasjes** [comment]
I did some checks on the other examples I have.
I think this is the best direction:
- Robot if the first version is nonsense
  - Safari/8vGeL2cj-43
- Normal browser with anti fingerprinting tool if the second version is nonsense
  - Safari/537.36/8vGeL2cj-43

**Comment by nielsbasjes** [comment]
The examples you provided are no longer Robots:
https://github.com/nielsbasjes/yauaa/blob/8e8afdd3c9a0f3684e3022390c94f56c6edc2fc2/analyzer/src/main/resources/UserAgents/Anonymized-Tests.yaml#L1358

To make the manipulation clear it shows this in these cases:
```
RemarkablePattern                    : 'Not a version/Anti fingerprinting'
```

**Comment by nielsbasjes** [comment]
I have released 7.9.1 with this fix.

**Comment by evelzi** [comment]
Excellent, thanks so much..!
